packages/inspector/src/inspector.ts

```diff
@@ -97,6 +104,13 @@ export class InspectorPanel extends Panel implements IInspector {
     this.addClass(PANEL_CLASS);
   }
 
+  /**
+   * Print in iframe
+   */
+  [printSymbol]() {
+    printWidget.bind(this)('.p-mod-hidden {display: none;}');
```

Comment:
Is there a way to grab more CSS so that this is less ad hoc? I admit this is stretching the bounds of my DOM API knowledge, but if we could do something like `node.getAllCSSAsString()`, that would be much nicer.